drivers/volume/csi/csi.go

```diff
@@ -670,13 +653,6 @@ func (c *csi) GetBackupStatus(backup *storkapi.ApplicationBackup) ([]*storkapi.A
 			return nil, err
 		}
 		log.ApplicationBackupLog(backup).Debugf("finished and uploaded %v snapshots and %v snapshotcontents", len(vsMap), len(vsContentMap))
-
```

Comment:
With this new change we are only cleaning up resources if a backup fails. But in case of CSI we clean up event after a successful backup. Same for snapshotsForRestore